client/asset/eth/eth.go

```diff
@@ -689,7 +697,7 @@ func (eth *ExchangeWallet) Swap(swaps *asset.Swaps) ([]asset.Receipt, asset.Coin
 		return nil, nil, 0, fmt.Errorf("unfunded contract. %d < %d", totalInputValue, totalSpend)
 	}
 
-	tx, err := eth.node.initiate(eth.ctx, swaps.Contracts, swaps.FeeRate, swaps.AssetVersion)
+	tx, err := eth.node.initiate(eth.ctx, swaps.Contracts, 200 /*swaps.FeeRate*/, swaps.AssetVersion)
```

Comment:
Here and in redeem, this was too low. Will fix this better.

Reply:
I'm not sure how to fix this. For one there is #1336 which is still my opinion. Just use the user's max fee. There is also #1372 which is a book.

I think I'll leave it in this unfinished state for this pr and let that get solved in another pr.

Reply:
I'm of the opinion that the server should just set the `Match.{FeeRateBase,FeeRateQuote}` to the `(dex.Asset).MaxFeeRate` for eth at match time. The current base fee rate is not relevant to the validation of the transaction. In theory, the client could just log the `MaxFeeRate` at order time and persist that info and use it as an input to the `Swap` method. But I don't see how that makes things easier for anyone. Instead, we can just think of the `Match.{FeeRateBase,FeeRateQuote}` as the "validation rate", or the rate that we are going to validate in their transaction. This fits into our current architecture without changes, and the relevant values are persisted.

Reply:
Agree completely with that @buck54321.  Ref, your comment in my issue on the matter: https://github.com/decred/dcrdex/issues/1372#issuecomment-1002273296

Reply:
Will leave for not if that's ok.

Reply:
OK, when we implement the changes discussed here and https://github.com/decred/dcrdex/issues/1372#issuecomment-1002273296 we will switch this back to `swaps.FeeRate`